src/process.rs

```diff
@@ -130,7 +171,7 @@ impl Process {
         }
 
         let process = TemporaryProcess(
-            Command::new(launch_options.path)
+            Command::new(&launch_options.path)
```

Comment:
Will this panic on systems where the binary doesn't exist? I'm not sure we ever want to let users construct a `LaunchOptions` where `path` is `None`.

Reply:
`path` can't be `None` here. If you have a `LaunchOption`, you have _some_ `path`. That's why `LaunchOption` no longer implements `Default`: In order to be able to return `None` if there is no default.

Reply:
oh, I see, thanks